src/sidebar/components/AutocompleteList.tsx

```diff
@@ -4,30 +4,45 @@ import { useMemo } from 'preact/hooks';
 
 import MenuArrow from './MenuArrow';
 
-/**
- * @template T
- * @param {T} item
- */
-const defaultListFormatter = item => item;
+const defaultListFormatter = <Item,>(item: Item) => item;
 
-/**
- * @template T
- * @typedef AutocompleteListProps
- * @prop {number} [activeItem] - The index of the highlighted item.
- * @prop {string} [id] - Optional unique HTML attribute id. This can be used
- *   for parent `aria-controls` coupling.
- * @prop {string} [itemPrefixId] - Optional unique HTML attribute id prefix
- *   for each item in the list. The final value of each items' id is
- *   `{itemPrefixId}{activeItem}`
- * @prop {T[]} list - The list of items to render. This can be a simple
- *   list of strings or a list of objects when used with listFormatter.
- * @prop {(item: T, index?: number) => any} [listFormatter] - An optional formatter
- *   to render each item inside an <li> tag This is useful if the list is an array of
- *   objects rather than just strings.
- * @prop {(item: T) => void} onSelectItem - Callback when an item is clicked with
- *   the mouse.
- * @prop {boolean} [open] - Is the list open or closed?
- */
+export type AutoCompleteListProps<Item> = {
```

Comment:
```suggestion
export type AutocompleteListProps<Item> = {
```

To match the component name.